xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1000,11 +1000,11 @@ bool CXBMCApp::GetStorageUsage(const std::string &path, std::string &usage)
   fmt << std::fixed;
   fmt.precision(1);
   fmt.width(PATH_MAXLEN);  fmt << std::left  << (path.size() < PATH_MAXLEN-1 ? path : StringUtils::Left(path, PATH_MAXLEN-4) + "...");
-  fmt.width(12);  fmt << std::right << totalSize << "G"; // size in GB
-  fmt.width(12);  fmt << usedSize << "G"; // used in GB
-  fmt.width(12);  fmt << freeSize << "G"; // free
+  fmt.width(11);  fmt << std::right << totalSize << "G"; // size in GB
+  fmt.width(11);  fmt << usedSize << "G"; // used in GB
+  fmt.width(11);  fmt << freeSize << "G"; // free
```

Comment:
assume that calling `fmt.width(11)` three times is mandatory, i suggest this formatting
```suggestion
  fmt.width(11);
  fmt << std::right << totalSize << "G"; // size in GB
  
  fmt.width(11);
  fmt << usedSize << "G"; // used in GB
  
  fmt.width(11);
  fmt << freeSize << "G"; // free
```